### PR TITLE
ao: Add version 6.9.0

### PR DESCRIPTION
**Pull request by TheRandomLabs**
Ao is an elegant Microsoft To Do desktop app. I wasn't sure if I should wrap the installer script in `architecture`.

### CODE REVIEW

File: bucket/ao.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "license": "MIT",
+    "url": "https://github.com/klaussinani/ao/releases/download/v6.9.0/ao-6.9.0.exe/#dl.7z",
+    "hash": "171a29295264c16e11bb35ae31326d92a81bc77a35561a71fafbc192224c855c",
+    "extract_dir": "$PLUGINSDIR",
+    "installer": {
+        "script": [
+            "Expand-7zipArchive \"$dir\\app-$(if ($architecture -eq '64bit') { '64' } else { '86' }).7z\" \"$dir\"",
+            "Remove-Item \"$dir\\app-*.7z\""
+        ]
+    },
```
[the remark this answers is not in the document]
Comment (TheRandomLabs) [reply bucket/ao.json]: I think I ran into problems with the NSIS installers, but I'll test again.

[the remark this answers is not in the document]
Comment (TheRandomLabs) [reply bucket/ao.json]: Never mind, I was thinking of the wrong thing. I'll do that.

File: bucket/ao.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "6.9.0",
+    "description": "Elegant Microsoft To Do desktop app.",
+    "homepage": "https://klaussinani.tech/ao/",
+    "license": "MIT",
+    "url": "https://github.com/klaussinani/ao/releases/download/v6.9.0/ao-6.9.0.exe/#dl.7z",
```
Comment (Ash258) [inline bucket/ao.json]:
```suggestion
    "url": "https://github.com/klaussinani/ao/releases/download/v6.9.0/ao-6.9.0.exe#/dl.7z",
```

File: bucket/ao.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        [
+            "Ao.exe",
+            "Ao"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/klaussinani/ao"
+    },
+    "autoupdate": {
+        "url": "https://github.com/klaussinani/ao/releases/download/v$version/ao-$version.exe/#dl.7z"
```
Comment (Ash258) [inline bucket/ao.json]:
```suggestion
        "url": "https://github.com/klaussinani/ao/releases/download/v$version/ao-$version.exe#/dl.7z"
```

**Pull request merged**